packages/inter-protocol/src/contractSupport.js

```diff
@@ -159,6 +159,7 @@ export const makeEphemeraProvider = init => {
 harden(makeEphemeraProvider);
 
 /**
+ * @see assertDefined for a static alternative
```

Comment:
Warn about the inequivalence. `assertDefined` tests for `=== undefined`, as it should.

`assertKeysDefined` test for falsiness, which itself is almost certainly a mistake.

Reply:
Good point. I've removed `assertKeysDefined` entirely.